Bpor.Rms/Bpor.Rms.Web/Startup/ConfigurationSetup.cs

```diff
@@ -19,4 +21,13 @@ public static void AddApplicationConfiguration(this ConfigurationManager configu
                 () => new AmazonSecretsManagerClient(RegionEndpoint.GetBySystemName(secretsManagerSettings.Region)));
         }
     }
+
+    public static void ConfigureDatabase(this IServiceCollection services, IConfiguration configuration)
+    {
+        var dbSettings = configuration.GetSection(DbSettings.SectionName).Get<DbSettings>();
```

Comment:
I think we'll need to call `Validate` on these settings obtained directly.

Can you create an extension method to:

1. Get the settings directly as above (and return them later).
2. Register the same settings as `IOptions`. They might be useful elsewhere in the application.
3. Validate these settings using the built in methods.

That extension will form part of our reusable library across the projects.